app/kafka/consumer.py

```
@@ -0,0 +1,113 @@
+
+
+    for message in consumer:
+        try:
+            event = order_events_pb2.OrderEvent()
+            event.ParseFromString(message.value)
+
+            if event.event_type == order_events_pb2.ORDER_CREATED:
+                process_order_created(event.order_created)
+        except DecodeError as e:
+            print("Error al decodificar mensaje:", e)
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)